test/e2e/framework/util.go

```diff
@@ -3137,7 +3137,7 @@ func WaitForDeploymentRevisionAndImage(c clientset.Interface, ns, deploymentName
 	var deployment *extensions.Deployment
 	var newRS *extensions.ReplicaSet
 	var reason string
-	err := wait.PollImmediate(Poll, 1*time.Minute, func() (bool, error) {
+	err := wait.Poll(Poll, 1*time.Minute, func() (bool, error) {
```

Comment:
We don't want to use events in tests 
https://github.com/kubernetes/kubernetes/issues/36453